cypress/integration/sidebar.spec.js/sidebar.spec.js

```diff
@@ -0,0 +1,95 @@
+/// <reference types="cypress" />
```

Comment:
what is this comment?

Reply:
this is standard in cypress files to allow for types. when you do cy. it will auto complete